### PR TITLE
gcc-4.8: require isl >=0.10,<=0.14

### PR DESCRIPTION
**Pull request by frol**
Only 0.10, 0.11, 0.12, 0.14 versions are checked by ./configure:

```
checking for version 0.10 of ISL... no
checking for version 0.11 of ISL... no
checking for version 0.12 of ISL... no
checking for version 0.14 of ISL... no
configure: error: Unable to find a usable ISL.  See config.log for details.
```

/cc @stuarteberg @msarahan

### CODE REVIEW

File: gcc-4.8/meta.yaml [hunk 1]
```diff
@@ -17,14 +17,14 @@ requirements:
...
     - mpfr >=2.4.2
     - mpc >=0.8.1
-    - isl
+    - isl >=0.10,<=0.14
     - cloog 0.18.0
     # Do not make gcc a build dependency (you will need to add it to the PATH manually)
   run:
-    - gmp >=4.2
-    - mpfr >=2.4.0
-    - mpc >=0.8.0
```
Comment (ccordoba12) [inline gcc-4.8/meta.yaml]: Why these changes in the run deps? Are they required by `isl`?

Comment (frol) [inline gcc-4.8/meta.yaml]: @ccordoba12 How much sense does it make using older run dependencies than build dependencies?

Comment (ccordoba12) [inline gcc-4.8/meta.yaml]: Ok, sorry. I didn't notice that. Merging then.
